draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -1574,8 +1577,8 @@ Example-Field: Baz
 <t>
    contains two field lines, both with the field name "Example-Field". The
    first field line has a field line value of "Foo, Bar", while the second
-   field line value is "Baz". The field value for "Example-Field" is a list
-   with three members: "Foo", "Bar", and "Baz".
+   field line value is "Baz". The field value for "Example-Field" is the list
+   "Foo, Bar, Baz".
```

Comment:
Giving an example of the actual list structure as it appears on the wire seems more useful to me than enumerating the list members individually.

Reply:
In retrospective, this is IMHO incorrect.